mimic/model/identity.py

```diff
@@ -0,0 +1,22 @@
+"""
+Models relating to identity.
+"""
+from attr import attributes, attr, validators
```

Comment:
We're all still trying to figure out the process a little bit :-). Thanks for the umbrella issue.